sp800-63c/sec6_presentation.md

```diff
@@ -6,6 +6,8 @@ Assertions MAY be presented in either a *back-channel* or *front-channel* manner
 
 The IdP SHALL transmit only those attributes that were explicitly requested by the RP. RPs SHALL conduct a privacy risk assessment when determining which attributes to request. 
 
+To mitigate the risk of unauthorized exposure of sensitive information (e.g., shoulder surfing), the CSP SHALL, by default, mask sensitive information displayed to the subscriber.  The CSP SHALL provide mechanisms for the subscriber to temporarily unmask such information in order for the subscriber to view full values.
```

Comment:
I'll add something. We also have the UX chapter coming, that will have more language about masking.